Cargo.toml

```diff
@@ -108,7 +108,8 @@ members = [
 
 # NOTE: These members should never include crates that require fuzzing
 # features or test features. These are the crates we want built with no extra
-# test-only code included.
+# test-only code included. These are essentially the main libra release
+# binaries.
 default-members = [
```

Comment:
So my understanding is that "fuzzing" feature should only appear as a "dev" dependency of any of these crates or any of their dependencies.  @sunshowers is this something guppy could check, or better enforce?

Reply:
Except for test binaries like cluster-test, which may take direct dependencies on fuzzing etc.